src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt

```diff
@@ -444,15 +444,17 @@ enum class InmateRepositorySql(val sql: String) {
         OMT.COMMENT_TEXT,
         OMT.START_DATE,
         OMT.END_DATE,
-        OMT.AGY_LOC_ID AS AGENCY_ID
+        OMT.AGY_LOC_ID AS AGENCY_ID,
+        AL.DESCRIPTION AS AGENCY_DESCRIPTION
         FROM OFFENDER_MEDICAL_TREATMENTS OMT
         JOIN OFFENDER_HEALTH_PROBLEMS OHP
-        ON OHP.OFFENDER_HEALTH_PROBLEM_ID = OMT.OFFENDER_HEALTH_PROBLEM_ID
-                AND OHP.START_DATE <= sysdate
-                AND (OHP.END_DATE >= sysdate or OHP.END_DATE is null)
+          ON OHP.OFFENDER_HEALTH_PROBLEM_ID = OMT.OFFENDER_HEALTH_PROBLEM_ID
+                  AND OHP.START_DATE <= sysdate
+                  AND (OHP.END_DATE >= sysdate or OHP.END_DATE is null)
         JOIN OFFENDER_BOOKINGS OBT
-        ON OBT.OFFENDER_BOOK_ID = OHP.OFFENDER_BOOK_ID
-                JOIN REFERENCE_CODES RC ON OMT.TREATMENT_CODE = RC.CODE AND RC.DOMAIN = 'HEALTH_TREAT'
+          ON OBT.OFFENDER_BOOK_ID = OHP.OFFENDER_BOOK_ID
+                  JOIN REFERENCE_CODES RC ON OMT.TREATMENT_CODE = RC.CODE AND RC.DOMAIN = 'HEALTH_TREAT'
+        LEFT OUTER JOIN AGENCY_LOCATIONS AL ON AL.AGY_LOC_ID = OMT.AGY_LOC_ID
```

Comment:
have to `left outer` here as column is nullable in `OFFENDER_MEDICAL_TREATMENTS` and there are 2,000 null rows in production